### PR TITLE
Conversions: Using unescaped HTML handlebars

### PR DESCRIPTION
**Pull request by mintsoft**
@GuiltyDolphin I can't really test it from here as I can't get to my instance of `duckpan server`; however this should resolve the HTML issue (https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2044#issuecomment-194817807).

cc @moollaza 

----------
IA Page: http://duck.co/ia/view/conversions

### CODE REVIEW

File: share/goodie/conversions/content.handlebars [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class='zci__conversions-controls'>
-    <h3 class='zci__caption zci__result'>{{styled_output}} {{right_unit}}</h3>
+    <h3 class='zci__caption zci__result'>{{{styled_output}}} {{right_unit}}</h3>
     <h4 class='zci__subheader'>
         <span class='zci__operation tx-clr--lt2'>Convert:</span>
         <span class='zci__input'>{{markup_input}} {{left_unit}}</span>
```
Comment (GuiltyDolphin) [inline share/goodie/conversions/content.handlebars]: Might we want to do it with this too? I see the `<sup>`s in the formatted input. Unless of course we can't guarantee this to be safe.

**Comment by mintsoft** [comment]
@GuiltyDolphin you are correct; now amended. The input is safe because it has to have parsed through the `numbersrole`, which is very well defined in format

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks, looks good locally! :dolphin:

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@GuiltyDolphin you _just_ beat me to it!

**Comment by GuiltyDolphin** [comment]
@moollaza :stuck_out_tongue:

**Comment by mintsoft** [comment]
Thanks @moollaza @GuiltyDolphin

**Pull request merged**